src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1405,29 +1492,112 @@ const cleanupSubscriptions = () => {
+
+Take for example the following GraphQL schema:
+
+```graphql
+type Post @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  title: String!
+  content: String
+  comments: [Comment] @hasMany
+}
+
+type Comment @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  content: String
+}
+```
+
+By default, subscriptions will be created for the following mutations:
+
+```graphql
+# Post type
+onCreatePost
+onUpdatePost
+onDeletePost
+
+# Comment type
+onCreateComment
+onUpdateComment
+onDeleteComment
+```
+
+One operation that is not covered is if you wanted to only subscribe to comments for a specific post.
+
+Because the schema has a one to many relationship enabled between posts and comments, you can use the auto-generated field `postCommentsId` that defines the relationship between the post and the comment to set this up in a new Subscription definition.
+
+To implement this, you could update the schema with the following:
+
+```graphql
+type Post @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  title: String!
+  content: String
+  comments: [Comment] @hasMany
+}
+
+type Comment @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  content: String
+  postCommentsId: ID!
+}
+
+type Subscription {
+  onCommentByPostId(postCommentsId: ID!): Comment
+    @aws_subscribe(mutations: ["createComment"])
+}
+
+```
 
-[comment]: # (Explanation of how to unsubscribe and when to for reducing redundancy with steps for these actions.)
+Now you can create a custom subscription for comment creation with a specific post id:
 
-You can unsubscribe from events by using subscriptions, a GraphQL feature that allows the server to send data to its clients when a specific event happens. You can enable real-time data integration in your app with a subscription.
-[question]: (RENE, changes OK in line 1413?)
+```js
+import { API } from 'aws-amplify';
+import { onCommentByPostId } from './graphql/subscriptions';
+
+API.graphql({
+  query: onCommentByPostId,
+  variables: {
+    postCommentsId: "12345"
+  }
+})
+.subscribe({
+  next: data => {
+    console.log('data: ', data)
+  }
+})
+```
+
+</Accordion>
+
+### Unsubscribe from a subscription
+
+You can also unsubscribe from events by using subscriptions by implementing the following:
 
 ```ts
 // Stop receiving data updates from the subscription
 sub.unsubscribe();
 ```
 
-### Recap
-You now have set up subscriptions for real-time events and understand how to filter and cancel these subscriptions when needed.
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
+**Recap:** You now have set up subscriptions for real-time events and understand how to filter and cancel these subscriptions when needed.
```

Comment:
Any tie-in to real-world use and application of the topic discussed OR reiterate the benefit to add here?